Yank/pipeline.py

```diff
@@ -518,6 +518,24 @@ def read_system_files(positions_file_path, parameters_file_path, system_options,
     'spce': 'SPCBOX',
 }
 
+# Map the OpenMM-style name for solvent to the tleap
+# name for a list of files which would enable the
+# solvent model to work. Servers as error checking,
+# but is not foolproof
+
+_OPENMM_LEAP_SOLVENT_FILES_MAP = {
+    'tip3p': 'leaprc.water.tip3p',
+    'tip3pfb': 'leaprc.water.tip3p',
+    'tip4pew': 'leaprc.water.tip4pew',
+    'tip5p': 'leaprc.water.tip4pew',  # Enables the EP atom type
+    'spce': 'leaprc.water.spce',
+}
+
+# Reverse map which helps in some cases
+_OPENMM_LEAP_SOLVENT_FILES_MAP_R = {
```

Comment:
Only the keys of this dictionary are used in the code. What about generating this at runtime in `_setup_system()` as `known_solvent_files  = set(_OPENMM_LEAP_SOLVENT_FILES_MAP.values())`?